docs/public/favicon/site.webmanifest

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Is there any reason that we should have a manifest? If I look at docs of other libraries, most of them omit it.

Reply:
It's good to have if users view the documentation while they are offline, but otherwise it doesn't do much.

Reply:
Yes please.